tests/test_docker.py

```diff
@@ -980,3 +818,20 @@ def container_field_test_boiler_plate(resp):
     del docker_container['Status']
     del fields['dockerIp']
     docker_container = _sort_ports(docker_container)
+
+
+def instance_activate_common_validation(resp):
```

Comment:
Take a look at the function right above this (```container_field_test_boiler_plate```)
Looks like your function should call that function and then do just a few more things on top of it